<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1109</width>
    <height>861</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1100</width>
     <height>850</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/* Arri√®re-plans */

QWidget#widget{
  background-color: qlineargradient(spread:pad, x1:0, y1:0.5, x2:1, y2:0.5, stop:0.1 #757F9A, stop:1 #D7DDE8);
}
QLabel#arriere_plan_choix{
background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0.3 rgba(146, 141, 171,100), stop:1 rgba(31, 28, 44,100));
border-radius:30px;
}
QLabel#arriere_plan_apercu{
background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0.3 rgba(146, 141, 171,80), stop:1 rgba(31, 28, 44,80));
border-radius:30px;
}
QLabel#arriere_plan_compte{
background-color:rgba(0,0,0,20);
border-top-right-radius:30px;
border-bottom-right-radius:30px;
}
QLabel#arriere_plan_liste{
background-color:rgba(0,0,0,20);
border-top-left-radius:30px;
border-bottom-left-radius:30px;
}



/* Labels */

QLabel{
color:#ffffff;
font: 25px &quot;SF Pro Regular&quot;;
qproperty-alignment: AlignCenter;
}
QLabel#label_titre{
font: 40px &quot;SF Pro Semibold&quot;;
 }
QLabel#label_liste{
font: 30px &quot;SF Pro Semibold&quot;;
qproperty-alignment: &quot;AlignHLeft | AlignVCenter&quot;;
}
QLabel#label_choix_compte{
font: 30px &quot;SF Pro Semibold&quot;;
qproperty-alignment: AlignCenter;
}


/* Boutons */

QPushButton {
  border:0px;
  color: rgba(255,255,255,1);
  border-bottom: 5px solid rgba(255,255,255,1);
  font: 30px &quot;SF Pro Regular&quot;;
  text-align: left;
  background-color: none;
}
QPushButton:pressed {
  border-bottom: 8px solid rgba(255,255,255,50);
}
QPushButton:hover:!pressed {
  border-bottom: 8px solid rgba(255,255,255,1);
}
QPushButton#pushButton_deco {
  font-size:20px;
}

/* Combobox */

QComboBox{
background-color:transparent;
border:0px;
border-bottom: 5px solid rgba(255,255,255,1);
color:rgba(255,255,255,1);
font: 20px &quot;SF Pro Regular&quot;;
}

QComboBox QAbstractItemView {
    background-color:#ffffff;
    selection-background-color:#fd314d;
    color:#000000;
    outline: none;
}
</string>
   </property>
   <widget class="QLabel" name="label_titre">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>1041</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Bienvenue User&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>440</y>
      <width>441</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Voici un aper√ßu de votre compte</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_comptes">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>350</y>
      <width>431</width>
      <height>61</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Acc√©der √† vos comptes ü°•</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_budget">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>480</y>
      <width>431</width>
      <height>62</height>
     </rect>
    </property>
    <property name="text">
     <string>G√©rer vos budgets ü°•</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_deco">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>761</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>‚Üê D√©connexion</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_liste">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>250</y>
      <width>441</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Souhaitez-vous :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_solde">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>500</y>
      <width>441</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Aucun compte.
Veuillez en cr√©er un gr√¢ce √† la page
des comptes.</string>
    </property>
   </widget>
   <widget class="QLabel" name="arriere_plan_choix">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>1041</width>
      <height>781</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_compte">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>350</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_choix_compte">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>441</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Choisissez un compte :</string>
    </property>
   </widget>
   <widget class="QLabel" name="arriere_plan_compte">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>431</width>
      <height>351</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="arriere_plan_liste">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>150</y>
      <width>541</width>
      <height>561</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>151</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:none;
font: 45px &quot;Goudy Bookletter 1911&quot;;
color:rgba(255,255,255,1)</string>
    </property>
    <property name="text">
     <string>DEMJ.</string>
    </property>
   </widget>
   <zorder>arriere_plan_choix</zorder>
   <zorder>arriere_plan_liste</zorder>
   <zorder>arriere_plan_compte</zorder>
   <zorder>pushButton_comptes</zorder>
   <zorder>pushButton_budget</zorder>
   <zorder>label_liste</zorder>
   <zorder>label_solde</zorder>
   <zorder>label</zorder>
   <zorder>comboBox_compte</zorder>
   <zorder>label_choix_compte</zorder>
   <zorder>label_titre</zorder>
   <zorder>pushButton_deco</zorder>
   <zorder>label_2</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
